querybook/server/logic/tag.py

```diff
@@ -119,7 +120,7 @@ def create_table_tags(
         meta = {
             "type": tag.type,
             "tooltip": tag.description,
-            "color": tag.color,
+            "color": closest_color(tag.color),
```

Comment:
if we are assuming tag.color has format `#AABBCC` then can you add a comment in DataTag?